### DISCUSSION TITLE
スマートフォンからだと「編集する」が開けない

### DISCUSSION THREAD

**Discussion by Brutus03** [increments/qiita-discussions#551] [要望・アイデア・不具合]
ご担当者　様

タイトルの通りですがスマートフォンからだと、下書き一覧の画面から「編集する」が開けません。
以前は開けたような気がしましたが、仕様変わりましたでしょうか？

![IMG_4701](https://github.com/increments/qiita-discussions/assets/38848514/4747833f-55d0-4061-a12f-c81803198dbe)

**Comment by tanaka-asami415** [comment]
@Brutus03 
先日仕様変更を行いました！
https://qiita.com/release-notes/2023-10-05

モバイル版は該当の記事カードをタップしていただくと編集していただくことが可能です。
よろしければ、下記ヘルプもご参照ください🙇
https://help.qiita.com/ja/articles/qiita-post#part-b2177221f4576d6c

**Reply by Brutus03** [reply]
ご回答ありがとうございます。

> モバイル版は該当の記事カードをタップしていただくと編集していただくことが可能です。

恐縮ですが、個人的には凄い分かりにくかったのと、間違って「削除する」を押す可能性が増えるのではないかと思いました。
モバイルから操作するシーンはあまりないと思いますが、「削除する」が目立っているのでUI見直していただけると幸いです。

**Reply by tanaka-asami415** [reply]
@Brutus03 
ご意見ありがとうございます。
時期は未定になりますが、検討いたします！🙏

**Reply by tanaka-asami415** [reply]
@Brutus03 
お待たせしました！こちら修正が完了しました✨
ご意見いただきありがとうございました！
https://qiita.com/release-notes/2023-12-14

**Reply by Brutus03** [reply]
修正確認しました。
クローズで大丈夫です。ご対応ありがとうございました。
